src/main/kotlin/com/terraformation/backend/support/atlassian/AtlassianHttpClient.kt

```diff
@@ -0,0 +1,51 @@
+package com.terraformation.backend.support.atlassian
+
+import com.terraformation.backend.config.TerrawareServerConfig
+import com.terraformation.backend.support.atlassian.resource.AtlassianResource
+import io.ktor.client.HttpClient
+import io.ktor.client.engine.java.Java
+import io.ktor.client.plugins.auth.*
+import io.ktor.client.plugins.auth.providers.*
+import io.ktor.client.plugins.contentnegotiation.ContentNegotiation
+import io.ktor.client.plugins.defaultRequest
+import io.ktor.client.request.accept
+import io.ktor.client.request.request
+import io.ktor.http.ContentType
+import io.ktor.http.contentType
+import io.ktor.serialization.jackson.JacksonConverter
+import jakarta.inject.Named
+
+/** Submits API requests to interact with Atlassian services. */
+@Named
+class AtlassianHttpClient(private val config: TerrawareServerConfig) {
+  private val httpClient: HttpClient by lazy { createHttpClient() }
+
+  suspend fun <T> makeRequest(resource: AtlassianResource<T>): T {
+    val response = httpClient.request { resource.buildRequest(this) }
+    return resource.parseResponse(response)
+  }
```

Comment:
The Ktor client uses suspend functions, but in our application code we generally stick to synchronous functions. The `runBlocking` could live here instead of the callers having to do it.

My assumption is that we're not going to be making tons of concurrent requests such that we'd need to worry about the thread count exploding.